Source/JavaScriptCore/assembler/AbstractMacroAssembler.h

```diff
@@ -57,6 +57,8 @@ namespace DFG {
 struct OSRExit;
 }
 
+#define JIT_COMMENT(...) if (Options::dumpDisassembly()) { jit.comment(__VA_ARGS__); } else { (void) jit; }
```

Comment:
Let's take `jit` as a first parameter. Some place uses `m_jit`, some place uses `JIT`'s pointer.

Reply:
Also, let's use `UNLIKELY` for `Options::dumpDisassembly()`.
And let's wrap everything with `do { } while (0)` as a normal C macro's convention to prevent accidental `else` etc chaining.
```
#define JIT_COMMENT(jit, ...) \
    do { \
        if (UNLIKELY(Options::dumpDisassembly())) \
            (jit).comment(__VA_ARGS__); \
    } while (0)
```